src/config/index.ts

```diff
@@ -59,6 +59,9 @@ export const getTxServiceUriFrom = (safeAddress) =>
 export const getIncomingTxServiceUriTo = (safeAddress) =>
   `safes/${safeAddress}/incoming-transfers/`
 
+export const getNewTransactionsServiceUriFrom = (safeAddress) =>
```

Comment:
Lets rename it to `getAllTransactionsUriFrom`

Yes, it's new, but only for now. We should name everything in a way so we don't have to rename it in the future. In the past I made a mistake of naming a file new gas calculation functions `gasNew.ts` (https://github.com/gnosis/safe-react/blob/development/src/logic/safe/transactions/gasNew.ts) and now you need to spend time on renaming it.

and also let's include types

Reply:
done